nebula/Makefile

```diff
@@ -0,0 +1,105 @@
+# Copyright 2021 Stan Grishin (stangri@melmac.net)
+# This is free software, licensed under the MIT License.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nebula
+PKG_VERSION:=1.3.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Stan Grishin <stangri@melmac.net>
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/slackhq/nebula/archive/
+PKG_HASH:=b94fba0251a4a436e25b127d0b9bc0181b991631f1dc8e344b1c8e895b55375d
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/nebula-$(PKG_VERSION)
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+GO_PKG:=github.com/slackhq/nebula
+GO_PKG_LDFLAGS_X:=main.Build=$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/feeds/packages/lang/golang/golang-package.mk
+
+define Package/nebula/default
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=nebula
+	URL:=https://github.com/slackhq/nebula
+	DEPENDS:=$(GO_ARCH_DEPENDS)
```

Comment:
```suggestion
  SECTION:=net
  CATEGORY:=Network
  TITLE:=nebula
  URL:=https://github.com/slackhq/nebula
  DEPENDS:=$(GO_ARCH_DEPENDS)
```

Reply:
All the other Makefiles I've contributed to have tabs and not spaces. Has there been a change in this recently?

Reply:
I don't think there was a change looking in https://openwrt.org/docs/guide-developer/packages and https://openwrt.org/docs/guide-developer/helloworld/chapter3

Reply:
Thanks, I've updated formatting and force-pushed updates.